bin/variables/unload-environment-variables.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+echo 'unset IDAM_VERSION'
+
+# urls
+echo 'unset SERVICE_AUTH_PROVIDER_API_BASE_URL'
+echo 'unset IDAM_API_BASE_URL'
+echo 'unset CCD_DEFINITION_STORE_API_BASE_URL'
+
+# secrets
+echo 'unset CCD_API_GATEWAY_S2S_SECRET'
```

Comment:
Any reason why this doesn't clear all of the secrets? I.e. CCD_DEFINITION_STORE_S2S_SECRET this one is not cleared.